package.json

```diff
@@ -1,57 +1,33 @@
 {
-  "name": "site",
-  "version": "0.1.0",
+  "name": "vite-test",
   "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
```

Comment:
It makes sense to add test command here as well

Reply:
yes, any suggestions about what library to use for testing, and if there is need for checking coverage?

Reply:
Either jest, or vitest, both have same api